### PR TITLE
Ra dspdc 1270 logic bug

### PR DESCRIPTION
**Pull request by raaidbroad**
Have the logic working as desired now; I check if processing is needed inside the export-diff workflow instead of trying to pull it out from a previous workflow and reuse it in this one. Less dry? Yep. Does it work? Yep. Argo, man. https://argo.monster-dev.broadinstitute.org/workflows/clinvar/export-diff-5kfdr?tab=workflow

### CODE REVIEW

**Review by pshapiro4broad — APPROVED** [review]
Looks good, as far as I can tell

File: orchestration/templates/export-diff.yaml [hunk 1]
```diff
@@ -75,23 +93,14 @@ spec:
...
-                - name: pipeline-version
-                  value: {{ $pipelineVersion | quote }}
-          {{- $precedingDate := "{{tasks.get-preceding-date.outputs.result}}" }}
-
           ## Processes to run when the preceding date is present
           - name: date-present
-            dependencies: [get-preceding-date]
+            dependencies: [is-preceding-date-present, check-processing-history]
+            {{- if $versionIsPinned }}
+            when: {{ "({{tasks.is-preceding-date-present.outputs.result}} > 0) && ({{tasks.check-processing-history.outputs.result}} == 0)" }}
```
[the remark this answers is not in the document]
Comment (raaidbroad) [reply orchestration/templates/export-diff.yaml:100]: There isn't, but I figured writing `> 0` would be clearer for the reader to know that it is supposed to be greater haha

**Review by quazi-broad — APPROVED** [review]
Walked through the template logic and changes look good. Did you decide to move the notify-clingen piece out to export-diff since that template was already getting the same inputs?

**Comment by raaidbroad** [comment]
> Walked through the template logic and changes look good. Did you decide to move the notify-clingen piece out to export-diff since that template was already getting the same inputs?

@quazi-broad yep, exactly

**Comment by raaidbroad** [comment]
Final boolean change fixed it, merging now

**Pull request merged**